core/base/v7/inc/ROOT/TLogger.hxx

```diff
@@ -113,8 +113,16 @@ public:
 } // namespace Experimental
 } // namespace ROOT
 
+#if defined(_MSC_VER)
+#define LOG_PRETTY_FUNCTION __FUNCSIG__
```

Comment:
Can you prefix the macro with R__ do reduce the risk of clash with user macros?

Thanks,
Philippe.

Reply:
Done